### DISCUSSION TITLE
Accordion

### DISCUSSION THREAD

**Discussion by stevenjmesser** [stevenjmesser/design-system-discussion-poc#8] [Backlog]
Use this issue to discuss the [accordion component in the GOV.UK Design System](https://design-system.service.gov.uk/components/accordion/).

**Comment by stevenjmesser** [comment]
Hello, here are some opinions about the Accordion.

**Comment by stevenjmesser** [comment]
Great!

If I transfer an issue to a discussion, I can see existing comments on the discussion. However, I can’t see the first comment on the discussion.

That might not be a problem though as the first comment on a discussion would probably just duplicate the content on the guidance page.

**Comment by rivalee** [comment]
Some accordion opinions.

**Comment by stevenjmesser** [comment]
That is a fantastic insight!

**Comment by kellylee-gds** [comment]
Test comment
